examples/Demo/Shared/Pages/Panel/Examples/SimplePanel.razor

```diff
@@ -1,11 +1,13 @@
 ﻿@implements IDialogContentComponent<SimplePerson>
 
+<FluentDialogBody>
 <h3>Hello @Content.Firstname </h3>
```

Comment:
Could you increase the miter by one unit to the right (to align the Body container correctly)?